tests/cisco_ios/show_lldp_neighbors_detail/cisco_ios_show_lldp_neighbors_detail1.yml

```diff
@@ -17,6 +17,11 @@ parsed_sample:
     power_priority: ""
     power_source: ""
     power_requested: ""
+    manufacturer: ""
```

Comment:
Where is the test data of it parsing out this data?

Reply:
The manufacturer already existed in [this](https://github.com/networktocode/ntc-templates/blob/ef2e7d6e46da9de78627447cf4b6b530472d2381/tests/cisco_ios/show_lldp_neighbors_detail/cisco_ios_show_lldp_neighbors_detail3.raw#L35) raw output. I just updated to yml to assert it was being parsed correctly.